pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyDao.java

```diff
@@ -24,4 +28,16 @@
     @SqlQuery("select * from study_exit_request where user_id = :userId")
     @RegisterConstructorMapper(StudyExitRequest.class)
     Optional<StudyExitRequest> findExitRequestForUser(@Bind("userId") long userId);
+
+    @SqlQuery(
+            "select lc.iso_language_code from study_language sl inner join language_code lc on sl.language_code_id = lc.language_code_id"
+            + " inner join umbrella_study us on sl.umbrella_study_id = us.umbrella_study_id where us.guid = :studyGuid"
+    )
+    Set<String> getSupportedLanguageCodesByGuid(@Bind("studyGuid") String studyGuid);
```

Comment:
I recommend naming this `findSupportedLanguageCodesbyGuid`. Likewise for the method below. I believe naming things using the `find*` pattern is better than `get*` since those have connotations with getters.

Reply:
Agree, the `find` prefix reflects the intention better. We still have `get` methods throughout the code, it would be nice to rename them to make things consistent.